src/components/datasworn/ConditonMeter/ConditionMeter.tsx

```diff
@@ -86,6 +86,9 @@ export function ConditionMeter(props: ConditionMeterProps) {
         <Box
           id={id}
           component={onActionClick && !disabled ? ButtonBase : "div"}
+          aria-label={
+            onActionClick && !disabled ? t("datasworn.roll", "Roll") : undefined
+          }
```

Comment:
Everything else looks great, I'm just not positive on this one - does the aria-label on line 155 handle labeling this button, or do we need it directly on the button itself?

Reply:
I needed it directly on the button so that I could find it in the test - there is probably a better way to handle that, but yes there is an aria label on the button element as well as a nested element

Reply:
data-testid and `getByTestId` within the test should handle this correctly without adding the aria-label.

Reply:
done